src/app/_components/SlideOver.tsx

```diff
@@ -10,9 +15,9 @@ type SlideOverProps = {
 
 export function SlideOver({ open, setOpen, children }: SlideOverProps) {
   return (
-    <Transition.Root show={open} as={Fragment}>
+    <Transition show={open} as={Fragment}>
       <Dialog className="relative z-10" onClose={setOpen}>
-        <Transition.Child
+        <TransitionChild
```

Comment:
Do we still need Transition? Looking at the [docs](https://headlessui.com/react/dialog#adding-transitions), we can now use a transition `prop` and use `data-closed`, `data-enter` and `data-leave` with Tailwind classes.

I haven't tried it so maybe we still need it, just sharing :)

Reply:
Hmm yea I remember we tried that, but we came back to this solution, don't remember the reason tho